### PR TITLE
Closed streams correctly wherever possible ; Feature 1 ; Feature 2;

### PR DESCRIPTION
**Pull request by Zacorich**
Addition to main issue solved here:
https://github.com/otros-systems/otrosvfsbrowser/pull/14

### CODE REVIEW

File: OtrosLogViewer-app/src/main/java/pl/otros/logview/api/io/Utils.java [hunk 1]
```diff
@@ -209,13 +209,50 @@ public static void closeQuietly(FileObject fileObject) {
...
+        fileObject.getFileSystem().getFileSystemManager().
+                closeFileSystem(fileObject.getFileSystem());
         LOGGER.info(String.format("File %s closed", friendlyURI));
       } catch (FileSystemException ignore) {
         LOGGER.info(String.format("File %s is not closed: %s", friendlyURI, ignore.getMessage()));
       }
     }
   }
 
+  public static void closeQuietly2(LoadingInfo fileObject) {
```
Comment (otrebski) [inline OtrosLogViewer-app/src/main/java/pl/otros/logview/api/io/Utils.java]: Please change name to closeQueitly

**Comment by otrebski** [comment]
Project is using Java 1.8. Please use Try with resources.

**Comment by Zacorich** [comment]
Will change everything to try-with-resources here too.

**Comment by otrebski** [comment]
I'm using default Intellij formatter.

**Comment by Zacorich** [comment]
After further inspection your style is more likely: Tab size:2 , Indent: 2 , Continuous indent: 2, Spaces within array initializer braces. I think default is 4,4,8, no spaces within braces as I recall. Not a big deal you can just export your style from intellij event if it is default and push it somewhere on github so no questions from developers like me in future. Thx

**Comment by Zacorich** [comment]
Please review latest commit. Thx

**Comment by Zacorich** [comment]
Added code style config to latest commit - 97bb7efe9cbd9f5b3a9927c83d72f1dc64cac6fa

**Comment by Zacorich** [comment]
Please check new feature added with following commit - 94454e88a41f56488a6b682318b958d5520696ca

**Comment by Zacorich** [comment]
Please check another feature added with following commit - 0fdbba93d8bed735455344b494f4bc8626434a8e . Adds option to set monospaced font for message chunk of the "message detail". Needed for ASCII art or tables to be presented correctly. Thx

**Comment by Zacorich** [comment]
Sorry, I should have moved 2 features into own branches to have multiple pull requests instead of one. My bad. Though as commits are independent from one another code review shouldn't be a problem. Thx

**Comment by Zacorich** [comment]
codacy argues about code formatting style mostly. Can you improve an entrypoint for new developers to join this project as I currently find it too hard to merge to you master branch? As I mentioned earlier you code styling profile would be nice so at least codacy/pr will pass green... plus maybe list of anti-patterns and things you don't want to see in you project. Thx

File: OtrosLogViewer-app/src/main/java/pl/otros/logview/api/BaseLoader.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
  ******************************************************************************/
 package pl.otros.logview.api;
 
+import org.apache.commons.io.IOUtils;
```
Comment (otros-systems) [inline OtrosLogViewer-app/src/main/java/pl/otros/logview/api/BaseLoader.java]: @Zacorich Unused imports

**Pull request merged**